cmd/oceantv/broadcast.go

```diff
@@ -460,13 +460,6 @@ func saveBroadcast(ctx context.Context, cfg *BroadcastConfig, store datastore.St
 		return fmt.Errorf("could not put broadcast data in store: %w", err)
 	}
 
-	// Ensure that the CheckBroadcast cron exists.
```

Comment:
I'm thinking we could also remove this version of saveBroadcast (oceanbench has a different version that we keep); any operation on broadcast configs should use transactions now, so it's dangerous keeping this.

Reply:
I agree, I tried to briefly but it's still used in SetupSecondary and I wasn't sure how to handle how it wants to create another new config, not just modify the existing one. Perhaps another issue/PR.

Reply:
Oh right, OceanBroadcastManager.Save should handle creation of a new broadcast. So we can replace that call with that. But year, another PR sounds good.